src/app/components/userprofile/userprofile.component.html

```diff
@@ -0,0 +1,36 @@
+<div class="container">
+    <div class="row Credentials justify-content-center ">
+        <div class="text-center col-5">
+            <label class="h2"><strong>Username</strong></label>
+            <br>
+            <img class="imgProfile" [src]="imageProfile">
```

Comment:
avoid using abbreviations for your identifiers.. use `image-profile` instead of `imgProfile`